src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,19 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        Book book = new Book();
+
+        // initialize field values using setters or constructor
+        bookDao.create(book);
```

Comment:
The `Book` object is created but not initialized with any field values before being passed to `bookDao.create(book)`. Ensure that the necessary fields of the `Book` object are set using setters or a constructor before calling the `create` method. This will prevent potential issues related to incomplete data.

Reply:
ok, done